<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1355</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>640</y>
      <width>288</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>288</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>1011</width>
      <height>911</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Новый столбец</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1355</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
